README.md

```diff
@@ -6,7 +6,7 @@ Deli is a performance modeling tool, allowing you to understand and experiment
 with new designs at several orders of magnitude faster than wall-clock time.
 Specifically, Deli is designed to help you understand how long it takes for
 'jobs' or 'requests' to complete in your system. Deli borrows concepts and
-terminology from [queueing
+terminology from [queuing
```

Comment:
I'd like to leave the spelling as `queueing` (https://en.wikipedia.org/wiki/Queueing_theory). Would you mind removing this change to the diff, and I can merge the other spellfix?

Reply:
ok

Reply:
updated